Automattic/wp-calypso
# Jetpack Manage: Adding pagination to the sites v2 dashboard

Related to https://github.com/Automattic/jetpack-manage/issues/311

## Proposed Changes

* This PR adds pagination to the sites V2 dashboard.
* This is assumes a starting per page figure of 50, which was pre-inputted, but can be changed if needed.
* This PR also includes field label translations, although not related to the linked issue

## Testing Instructions

* To test pagination, connect multiple Jurassic Ninja sites to Jetpack (that's assuming you don't already have 10 or more sites in the Sites dashboard).
* You will also need to apply the WPcom patch at D140045-code, make sure you are sandboxed and your hosts file points to WPcom (until or unless that patch gets merged)
* Then with this PR applied, visit the Sites dashboard: `http://jetpack.cloud.localhost:3000/sites`
* From the 'View options' icon (top right) of the sites dashboard, select 10 rows per page. You should see 10 connected sites showing per page, and on selecting page 2 from the pagination options (bottom right), the remaining connected sites should show. If you have more than 20 sites, you should have more pagination options.
* If you then select 20 rows per page or a number higher than your number of sites, then all sites should display in the dashboard.

**Screenshots**

Pagination selector (if you don't have the development environment label and bug report badge in the way...)
<img width="399" alt="Screenshot 2024-02-28 at 13 19 42" src="https://github.com/Automattic/wp-calypso/assets/16754605/14abb5da-f6f7-46a0-9c12-6b9d157d292f">


## Pre-merge Checklist
